### ISSUE TITLE
Fixture specs flake under parallel load

### ISSUE THREAD

**Issue by christopher-buss** [christopher-buss/eslint-config#601]
## Summary

`test/fixtures.spec.ts` and `test/oxlint-fixtures.spec.ts` fail intermittently under full-suite parallel load and pass in isolation.

Observed once in 14 full-suite runs on the #599 branch:

```
 FAIL  test/fixtures.spec.ts > 'roblox-game' > should produce expected lint output
 FAIL  test/fixtures.spec.ts > 'package' > should produce expected lint output
 FAIL  test/oxlint-fixtures.spec.ts > oxlint standalone fixtures > should produce the expected diagnostics
 FAIL  test/oxlint-fixtures.spec.ts > oxlint standalone fixtures > should run the cspell jsPlugin end-to-end
 Test Files  2 failed | 17 passed (19)
      Tests  4 failed | 286 passed (290)
```

`vitest run test/oxlint-fixtures.spec.ts` in isolation: 2/2 pass. In the same session an ESLint child also exited `3221225477` (0xC0000005 access violation) while vitest ran in parallel; it did not recur.

## Likely mechanism

`_fixtures` is a gitignored, **shared, mutable** scratch directory that the fixture specs write into. It demonstrably accumulates content across runs — consecutive `tsc --noEmit` invocations reported a growing error set against `_fixtures/roblox-game/typescript.ts`:

```
run 1: 3 errors
run 2: 5 errors   (new: TS2300 Duplicate identifier 'join')
```

Deleting `_fixtures` and re-running gives a clean typecheck. A shared mutable directory plus parallel specs writing into it is a plausible race, and it fits the "fails under load, passes in isolation" signature.

Note this also means a `pnpm test` run leaves the tree in a state where `pnpm typecheck` fails, which is confusing on its own.

## Suggested direction

Give each fixture spec its own temp directory rather than sharing `_fixtures`, or serialise the specs that write into it.

## Not the cause

Not introduced by #599 — that change touches cache keying, not fixture linting. It may raise the failure rate by adding 4 tests that drive the TS builder over temp fixtures, increasing parallel load, but the underlying race predates it.
